samples/crypto/tls/Kconfig

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+source "Kconfig.zephyr"
+
+menu "PSA TLS sample"
+
+menuconfig TEST_RSA_CERTIFICATE_ENABLED
```

Comment:
Why `menuconfig` instead of `config`. With `menuconfig MY_CONFIG`, there's typically an

```
if MY_CONFIG
...
endif
```
Somewhere.

Reply:
I changed them to `config`.